### PR TITLE
Script to reverse the processed cosmo emissions onto a latlon grid

### PR DESCRIPTION
**Pull request by jmhaussaire**
This is a PR to be able to process a COSMO emission inventory back into a different grid.
The use case scenario for this is for instance:
You have EDGAR emissions, you process them onto your COSMO grid. Then, for instsance, you make some inverse modelling and get a different emission inventory in a similar file format than your original COSMO emissions. You may want to transfer that back into a standard lat/lon grid again. 

[config_latlon.txt](https://github.com/C2SM-RCM/cosmo-emission-processing/files/3949588/config_latlon.txt)

### CODE REVIEW

File: emiproc/grids.py [hunk 1]
```diff
@@ -511,8 +514,11 @@ def _LV03_to_WGS84(self, y, x):
 
 
 class COSMOGrid(Grid):
-    """Class to manage a COSMO-domain"""
-
+    """Class to manage a COSMO-domain
+    This grid is defined with a rotated pole coordinate system.
```
Comment (mjaehn) [inline emiproc/grids.py]: with -> as

**Comment by mjaehn** [comment]
Thanks for this PR. I think this is a very useful addition. Your implementation works fine. I was able to reproduce your example and also processed one of mines as well. Some minor comments:

- When processing the country map, I get several messages of this type:
```
The following countries were found, but didn't have a corresponding code
{'ISR', 'EGY', 'SAU'}
```
- The config variable `cosmo_grid` is now a bit misleading. We should consider renaming it to, for example, `output_grid`.

File: emiproc/grids.py (hunk 1 above)
Comment (jmhaussaire) [reply emiproc/grids.py]: done

**Comment by jmhaussaire** [comment]
- Regarding the first point:
Our country list is not exhaustive and the test domain that I pick was very large, so it is 'normal' that some countries are not found.
- Regarding the second point:
I agree with you. However, this would require changing a lot of areas in the code, with the risk of forgetting some, meaning that it would also require a lot of testing. It is not worth it in my opinion.

@gkuhl You are the only one who can merge. If you don't have any specific comments, it would be great if you could merge it !

**Comment by gkuhl** [comment]
Sorry, it seems I forgot this pull request. It's merged now.

**Pull request merged**